### PR TITLE
8276930: Update ProblemList

### PR DESCRIPTION
**Pull request by JesperIRL**
Update a few entries in the problemlist that refer to bugs that have been closed as duplicates.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8276930](https://bugs.openjdk.java.net/browse/JDK-8276930): Update ProblemList

### Reviewers
 * [Kevin Walls](https://openjdk.java.net/census#kevinw) (@kevinjwalls - Committer)
 * [David Holmes](https://openjdk.java.net/census#dholmes) (@dholmes-ora - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6328/head:pull/6328` \
`$ git checkout pull/6328`

Update a local copy of the PR: \
`$ git checkout pull/6328` \
`$ git pull https://git.openjdk.java.net/jdk pull/6328/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6328`

View PR using the GUI difftool: \
`$ git pr show -t 6328`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6328.diff">https://git.openjdk.java.net/jdk/pull/6328.diff</a>

</details>

### CODE REVIEW

**Review by kevinjwalls — APPROVED** [review]
Serviceability tidyup looks good to me.

File: test/jdk/ProblemList.txt [hunk 1]
```diff
@@ -787,7 +786,7 @@ sun/util/locale/provider/CalendarDataRegression.java            8268379 macosx-x
 
 # svc_tools
 
-sun/tools/jhsdb/BasicLauncherTest.java                          8211767 linux-ppc64,linux-ppc64le
+sun/tools/jhsdb/BasicLauncherTest.java                          8228649 linux-ppc64,linux-ppc64le
```
Comment (plummercj) [inline test/jdk/ProblemList.txt:789]: 8228649 is also resolved. Maybe this isn't an issue anymore. Would need to ask someone on the PPC team.

Comment (JesperIRL) [reply test/jdk/ProblemList.txt:789]: Yes, there were a couple of them that was duplicates of closed bugs, but this change only deals with duplicates. To actually remove the tests from the list I would need to run them for a while to make sure they pass now. That's a later task.

**Review by dholmes-ora — APPROVED** [review]
Hi Jesper,

Switching to the alternate bug id looks good.

Thanks,
David

**Pull request closed without merging**